src/Server.jl

```diff
@@ -736,13 +736,22 @@ function make_pull_request(pp::ProcessedParams, rp::RequestParams, rbrn::RegBran
     pr = nothing
     repo = join(split(target_registry["repo"], "/")[end-1:end], "/")
     msg = ""
+    auth = get_user_auth()
     try
-        pr = create_pull_request(repo; auth=get_user_auth(), params=params)
+        pr = create_pull_request(repo; auth=auth, params=params)
         msg = "created"
         @debug("Pull request created")
     catch ex
         if is_pr_exists_exception(ex)
-            @debug("Pull request already exists, not creating")
+            @debug("Pull request already exists, trying to update body")
+            prs = pull_request(repo; auth=auth, params=Dict(
+                "base" => target_registry["base_branch"],
+                "head" => split(repo, "/")[1] * ":" * brn,
```

Comment:
```suggestion
                "head" => params["head"],
```

Reply:
I don't think this does what we want, the GitHub docs seem to say that the value must also contain the user. https://developer.github.com/v3/pulls/#parameters